From 9c386911b16d4bf2f24e55ed42bfa397073b4b84 Mon Sep 17 00:00:00 2001
From: Johannes Berg <johannes.berg@intel.com>
Date: Thu, 14 Sep 2023 09:13:10 +0200
Subject: [PATCH] wifi: cfg80211: fix kernel-doc for wiphy_delayed_work_flush()
Git-commit: 9c386911b16d4bf2f24e55ed42bfa397073b4b84
Patch-mainline: v6.7-rc1
References: bsc#1227149

Now that I fixed the function name, we see the parameters
are wrong as well. Fix that too.

Change-id: I1a4cfea446875998a5a242ca36acc8244991a199
Reported-by: Stephen Rothwell <sfr@canb.auug.org.au>
Signed-off-by: Johannes Berg <johannes.berg@intel.com>
Acked-by: Takashi Iwai <tiwai@suse.de>

---
 include/net/cfg80211.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/net/cfg80211.h b/include/net/cfg80211.h
index a5841a96284d..8fcfe1869424 100644
--- a/include/net/cfg80211.h
+++ b/include/net/cfg80211.h
@@ -5954,7 +5954,7 @@ void wiphy_delayed_work_cancel(struct wiphy *wiphy,
 /**
  * wiphy_delayed_work_flush - flush previously queued delayed work
  * @wiphy: the wiphy, for debug purposes
- * @work: the work to flush
+ * @dwork: the delayed work to flush
  *
  * Flush the work (i.e. run it if pending). This must be called
  * under the wiphy mutex acquired by wiphy_lock().
-- 
2.43.0

